server/odc-service/src/main/java/com/oceanbase/odc/service/resourcehistory/ResourceLastAccessService.java

```diff
@@ -68,6 +69,18 @@ public ResourceLastAccessEntity add(Long organizationId, Long projectId, Long us
         return entity;
     }
 
+    public int batchDeleteResourceIds(Long organizationId, Long projectId, ResourceType resourceType,
```

Comment:
`@NonNull` annotation may be better?
why should we judge null if we have done it at the `server/odc-service/src/main/java/com/oceanbase/odc/metadb/resourcehistory/ResourceLastAccessRepository.java`